src/ReinforcementLearningCore/src/utils/networks.jl

```diff
@@ -255,8 +255,9 @@ Transform a vector containing the non-zero elements of a lower triangular da x d
 function vec_to_tril(cholesky_vec, da)
     batch_size = size(cholesky_vec, 3)
     c2idx(i, j) = ((2da - j) * (j - 1)) ÷ 2 + i #return the position in cholesky_vec of the element of the triangular matrix at coordinates (i,j)
+    softplusbeta(x) = log(exp(0.1f0 * x) +1)*10f0 #a softer softplus to avoid vanishing values
```

Comment:
Do we want to use a parameterized function which includes beta as an argument for softplusbeta? 

Something like `softplusbeta(x, β=100.f0) = 1f0 / β * log(1f0 + exp(β*x))` from this issue: https://github.com/FluxML/Flux.jl/issues/1216

Performance-wise there's no difference afaict.